pkg/cli/cvd.go

```diff
@@ -350,20 +351,30 @@ func (c *cvdCreator) createCVDFromLocalSrcs() ([]*hoapi.CVD, error) {
 	return res.CVDs, nil
 }
 
-func credentialsFactoryFromSource(source string) (CredentialsFactory, error) {
+func credentialsFactoryFromSource(source string, projectID string) (CredentialsFactory, error) {
 	switch source {
 	case NoneCredentialsSource:
-		return func() string { return "" }, nil
+		return func() client.BuildAPICredential {
+			return client.BuildAPICredential{AccessToken: "", UserProjectID: ""}
```

Comment:
you can avoid assigning fields with default values, `return client.BuildAPICredential{}` would work.